docs/examples/extract-saturation.py

```diff
@@ -0,0 +1,63 @@
+"""
+Extract layer saturation
+-------------------------------
+Extract layer saturation with delve.
+"""
+from os import mkdir
+from os.path import exists
+
+import torch
+from tqdm import trange
+
+from delve import CheckLayerSat
+
+
+class TwoLayerNet(torch.nn.Module):
+    def __init__(self, D_in, H, D_out):
+        super(TwoLayerNet, self).__init__()
+        self.linear1 = torch.nn.Linear(D_in, H)
+        self.linear2 = torch.nn.Linear(H, D_out)
+
+    def forward(self, x):
+        h_relu = self.linear1(x).clamp(min=0)
+        y_pred = self.linear2(h_relu)
+        return y_pred
+
+if not exists("regression/"):
+    mkdir("regression/")
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+torch.manual_seed(1)
+
+for h in [3, 32]:
+    # N is batch size; D_in is input dimension;
+    # H is hidden dimension; D_out is output dimension.
+    N, D_in, H, D_out = 64, 1000, h, 10
+
+    # Create random Tensors to hold inputs and outputs
+    x = torch.randn(N, D_in)
+    y = torch.randn(N, D_out)
+
+    model = TwoLayerNet(D_in, H, D_out)
+
+    x, y, model = x.to(device), y.to(device), model.to(device)
+
+    layers = [model.linear1, model.linear2]
+    stats = CheckLayerSat('regression/h{}'.format(h), save_to="plotcsv", modules=layers, device=device, stats=["lsat", "lsat_eval"])
+
+    loss_fn = torch.nn.MSELoss(size_average=False)
+    optimizer = torch.optim.SGD(model.parameters(), lr=1e-4, momentum=0.9)
+    steps_iter = trange(2000, desc='steps', leave=True, position=0)
+    steps_iter.write("{:^80}".format(
+        "Regression - TwoLayerNet - Hidden layer size {}".format(h)))
+    for _ in steps_iter:
+        y_pred = model(x)
```

Comment:
Whats your opinion on including some code snippet that just logs every second k-steps?

`stats.stop() if k%i != 0 else stats.resume()`
this would disable plotting and saturation computation for all but every kth-step and make the results more...digestable. It also gives you some performance boost depending on the network.
However it is not exactly something you will absolutely needs so it might be to much for a quick example.

Reply:
Do you mean something different than `log_interval`:
https://github.com/delve-team/delve/blob/7ae9125c705006d11a6d329fea8c27797757b19e/delve/torchcallback.py#L61

I'm in favor of cleaning the terminal output during running - maybe at least changing logging.info to `logging.debug`:
https://github.com/delve-team/delve/blob/7ae9125c705006d11a6d329fea8c27797757b19e/delve/writers.py#L515